integrations/phone/examples/common-interactions/README.md

```diff
@@ -0,0 +1,61 @@
+## Phone Starter Kit
+
+This example contains Watson Assistant phone integration actions that demonstrate how to:
+
+- How to use the phone and SMS channel for 2-factor authentication via the [send a text message](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-phone-actions#phone-actions-sms.)
```

Comment:
two-factor, not 2-factor

period on end of URL needs to come outside the parentheses (link...sms).

Reply:
```suggestion
- How to use the phone and SMS channel for two-factor authentication via the [send a text message](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-phone-actions#phone-actions-sms.)
```